src/main/resources/db/migration/0400/V400__PlotT0Density.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE tracking.t0_plot;
+
+CREATE TABLE tracking.plot_t0_density
+(
+    monitoring_plot_id BIGINT  NOT NULL REFERENCES tracking.monitoring_plots ON DELETE CASCADE,
+    species_id         BIGINT  NOT NULL REFERENCES species ON DELETE CASCADE,
+    plot_density       NUMERIC NOT NULL,
+    PRIMARY KEY (monitoring_plot_id, species_id)
+);
+
+CREATE TABLE tracking.plot_t0_observations
+(
+    monitoring_plot_id BIGINT NOT NULL REFERENCES tracking.monitoring_plots ON DELETE CASCADE,
+    observation_id     BIGINT NOT NULL REFERENCES tracking.observations ON DELETE CASCADE,
```

Comment:
Could add a foreign key constraint to make sure the plot is actually in the observation.
```sql
FOREIGN KEY (observation_id, monitoring_plot_id)
REFERENCES observation_plots (observation_id, monitoring_plot_id)
ON DELETE CASCADE
```